src/screens/SendReceive/Send/AssetsScreen/AssetRow.tsx

```diff
@@ -160,13 +152,14 @@ const AssetRow = ({ asset, style, isLast }: AssetRowProps) => {
                 onChangeText={handleOnAmountChange}
                 keyboardType="number-pad"
                 inputMode="decimal"
-                onBlur={handleInputBlur}
                 ref={inputRef}
+                multiline={true}
+                numberOfLines={2}
```

Comment:
When choosing "Use max" many times we get a very long number. This solution should be good enough to cover all cases.

Reply:
nice one.